common/windows/delphi/Makefile

```diff
@@ -4,6 +4,6 @@ global-versions:
 # Build our root version.txt; TODO: replace with build.sh deps
     cd $(KEYMAN_ROOT)\common\windows
     $(MKVER_U) $(KEYMAN_ROOT)\common\windows\delphi\general\keymanversion_build.in $(KEYMAN_ROOT)\common\windows\delphi\general\keymanversion_build.inc
-    $(GIT_BASH_FOR_KEYMAN) $(KEYMAN_ROOT)/common/include/build.sh clean configure build
+    $(GIT_BASH_FOR_KEYMAN) $(KEYMAN_ROOT)\common\include\build.sh clean configure build
```

Comment:
This change is unnecessary -- forward slashes are appropriate.

Reply:
That's what I thought as well, but forward slashes things didn't work when I tried from the command line. The problem is that `cmd.exe` doesn't like paths with forward slashes.

Reply:
OK, I can see that if the `GIT_BASH_FOR_KEYMAN` variable is not set we'll run into the problem you describe.

We should be requiring it for Windows builds, although it is currently documented as optional. We haven't really tested not having it, and I think we'll run into other problems such as non-linear builds (e.g. we need `start /wait` on this command without the variable defined; it also pops an awful bash window for each command). So it makes the builds much less reliable if it isn't present. 

So have you set your `GIT_BASH_FOR_KEYMAN` environment variable? See docs/build/windows.md for details.

I'd prefer not to be changing makefiles in this PR.

Reply:
Ah, ok. With `GIT_BASH_FOR_KEYMAN` things work, so I'll revert the changes to `Makefile`.